example-gto/code/mlem/requirements.txt

```diff
@@ -0,0 +1,3 @@
+gto
+emoji
```

Comment:
Q: why do we need this as a separate dependency?

Reply:
Thanks, forgot about it. Removing that.